context.go

```diff
@@ -222,15 +223,26 @@ func (c *Ctx) detectGOPATH(path string) (string, error) {
 // ImportForAbs returns the import path for an absolute project path by trimming the
 // `$GOPATH/src/` prefix.  Returns an error for paths equal to, or without this prefix.
 func (c *Ctx) ImportForAbs(path string) (string, error) {
-	srcprefix := filepath.Join(c.GOPATH, "src") + string(filepath.Separator)
-	if fs.HasFilepathPrefix(path, srcprefix) {
-		if len(path) <= len(srcprefix) {
+
+	gopathEvaluated, err := filepath.EvalSymlinks(c.GOPATH)
+	if err != nil {
+		return "", fmt.Errorf("Error evaluating symlinks in GOPATH %s: %s", c.GOPATH, err.Error())
```

Comment:
Errors should start with a lowercase letter.

Also, use `errors.Wrap` to achieve better results:
```go
errors.Wrapf(err, "failed to evaluate symlink %s", c.GOPATH)
```